### PR TITLE
Feature/tpi adjust date format

### PR DESCRIPTION
**Pull request by simaob**
Set the default date printing format.

### CODE REVIEW

File: config/initializers/date_time.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Date
+Date::DATE_FORMATS[:default] = '%d/%m/%Y'
```
Comment (tsubik) [inline config/initializers/date_time.rb]: We already have time_date_formats.rb intializer, so better to add that there. Didn't know that this is the way to change defualt format.

**Comment by tsubik** [comment]
When adding new event the date selected by datepicker is still in old format, when editing existing one, it's in the new format. There must be some config for datepicker format.

**Comment by tsubik** [comment]
I think I have an idea how to quickly change the default datepicker format for all date pickers.

File: config/initializers/date_time.rb (hunk 1 above)
Comment (simaob) [reply config/initializers/date_time.rb]: I'll move it!

**Comment by simaob** [comment]
I see you did it! Thanks =)

**Pull request merged**